ws/src/main/scala/de/thm/ii/submissioncheck/misc/LDAPConnector.scala

```diff
@@ -1,25 +1,69 @@
 package de.thm.ii.submissioncheck.misc
 
-import org.ldaptive.{DefaultConnectionFactory, LdapEntry, SearchExecutor}
+import java.util.Properties
+import javax.naming.Context
+import javax.naming.directory.InitialDirContext
+import org.ldaptive._
 
 /**
   * Using ldaptive to simple register to THM LDAP Service
+  *
   * @author http://www.ldaptive.org/
   */
-object LDAPConnector
-{
+object LDAPConnector {
+  private val LDAPURL = "ldaps://ldap.fh-giessen.de"
```

Comment:
Can you move the real URL into the resources/application.properties file and read it from there, please.